### ISSUE TITLE
canonical isomeric smiles sometimes has explicit hydrogen

### ISSUE THREAD

**Issue by ChayaSt** [openforcefield/cmiles#4]
Canonical isomeric smiles sometimes has explicit hydrogens.

**Comment by davidlmobley** [comment]
Sometimes aren't explicit hydrogens necessary to indicate the chirality? (But maybe I'm confused).

**Comment by ChayaSt** [comment]
Yes. But sometimes all hydrogens are given explicitly, not just the ones needed for chirality.

**Comment by ChayaSt** [comment]
Addressed by #8

**Comment by ChayaSt** [comment]
Oops! With RDKit molecules, `mol_to_smiles(mol, explicit_hydrogen=False)` returns a SMILES with explicit hydrogen.
